src/api/app/controllers/webui/users/notifications_controller.rb

```diff
@@ -22,10 +22,14 @@ def update
                       fetch_notifications.where(id: params[:notification_ids])
                     end
     # rubocop:disable Rails/SkipsModelValidations
-    unless notifications.update_all('delivered = !delivered')
+    updated_notifications = notifications.update_all('delivered = !delivered')
+    # rubocop:enable Rails/SkipsModelValidations
+
+    if updated_notifications.blank?
```

Comment:
How will this happen? `update_all` returns always number I think.

```
[2] pry(main)> User.where(login: 'peter').count
=> 0
[3] pry(main)> User.where(login: 'peter').update_all(biography: 'franz')
=> 0
```

```
[4] pry(main)> User.where(login: 'hennevogel').count
=> 1
[6] pry(main)> User.where(login: 'hennevogel').update_all(biography: 'franz')
=> 1
```

Reply:
I think I had `empty?` and it was suggested to `blank?` when the right approach would be `zero?`